instrumentation-tests/src/main/res/raw/route_response_japan_1.json

```diff
@@ -0,0 +1 @@
],"weight":61.799,"out":1,"duration":58.856,"entry":[false,true],"is_urban":false,"admin_index":4,"geometry_index":1935,"location":[140.744637,36.889408],"mapbox_streets_v8":{"class":"motorway"}},{"bearings":[32,35,216],"in":2,"classes":["toll","motorway"],"turn_duration":0.018,"weight":4.23,"out":1,"duration":3.953,"entry":[true,true,false],"is_urban":true,"admin_index":4,"geometry_index":1964,"location":[140.748113,36.901249],"mapbox_streets_v8":{"class":"motorway"}},{"bearings":[35,215],"in":1,"classes":["toll","motorway"],"weight":1.845,"out":0,"duration":1.716,"entry":[true,false],"is_urban":true,"admin_index":4,"geometry_index":1965,"location":[140.748725,36.901938],"mapbox_streets_v8":{"class":"motorway"}},{"bearings":[34,215],"in":1,"classes":["toll","motorway"],"weight":16.698,"out":0,"duration":15.533,"entry":[true,false],"is_urban":true,"admin_index":4,"geometry_index":1966,"location":[140.748988,36.902244],"mapbox_streets_v8":{"class":"motorway"}},{"bearings":[25,205,215],"in":1,"classes":["toll","motorway"],"weight":3.829,"turn_duration":0.007,"out":0,"duration":2.871,"entry":[true,false,false],"is_urban":true,"turn_weight":0.75,"admin_index":4,"location":[140.751144,36.905226999999996],"geometry_index":1973,"mapbox_streets_v8":{"class":"motorway"}},{"bearings":[23,205],"in":1,"classes":["toll","motorway"],"weight":10.278,"out":0,"duration":9.789,"entry":[true,false],"is_urban":false,"admin_index":4,"geometry_index":1975,"location":[140.751475,36.905797],"mapbox_streets_v8":{"class":"motorway"}},{"bearings":[26,204],"in":1,"classes":["toll","motorway"],"weight":50.108,"out":0,"duration":48.886,"entry":[true,false],"is_urban":false,"admin_index":4,"geometry_index":1977,"location":[140.752556,36.907794],"mapbox_streets_v8":{"class":"motorway"}},{"bearings":[16,197],"in":1,"classes":["toll","motorway"],"weight":4.254,"out":0,"duration":4.254,"entry":[true,false],"is_urban":false,"admin_index":4,"geometry_index":1994,"location":[140.758764,36.917274],"mapbox_streets_v8":{"class":"motorway"}},{"bearings":[14,196],"in":1,"classes":["toll","motorway"],"weight":4.746,"out":0,"duration":4.746,"entry":[true,false],"is_urban":false,"admin_index":4,"geometry_index":1995,"location":[140.759085,36.918169],"mapbox_streets_v8":{"class":"motorway"}},{"bearings":[12,193],"in":1,"classes":["toll","motorway"],"weight":19.841,"out":0,"duration":19.841,"entry":[true,false],"is_urban":false,"admin_index":4,"geometry_index":1997,"location":[140.759392,36.919179],"mapbox_streets_v8":{"class":"motorway"}},{"bearings":[21,197],"in":1,"classes":["toll","motorway"],"weight":17.182,"out":0,"duration":17.182,"entry":[true,false],"is_urban":false,"admin_index":4,"geometry_index":2001,"location":[140.760646,36.923418],"mapbox_streets_v8":{"class":"motorway"}},{"bearings":[34,211],"in":1,"classes":["toll","motorway"],"weight":1.738,"out":0,"duration":1.738,"entry":[true,false],"is_urban":false,"admin_index":4,"geometry_index":2010,"location":[140.762697,36.926803],"m
```

Comment:
Could you please format it in AS? I've had cases like:
1) I want to modify input data a bit;
2) For that I format json in AS;
3) Then I modify the data.
But in this case my diff will be enormous. If you format it right away, changes are easily tracked later.
Same for other long files.

Reply:
I can do that in this case but in general I was trying to be mindful of clone and checkout times of the repository (as it happens on each CI run). This response is 4 times bigger prettified (more than 3MB) than it is when single-lined.

Reply:
We might want to look into git LFS at some point, especially when we finally get around to mocking routing tiles and other larger resources.

Reply:
Oh, really? I didn't think of that. It just becomes harder to track changes. Never mind then.